index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do App</title>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <h1 class="heading">To Do List</h1>
+    <button class="addCardBtn">Add Card </button>
+
+    <div class="popup">
+        <form class="addCard">
+            <i class="fas fa-times-circle closeBtn"></i>
+            <p class="addCardElement">Card name</p>
+            <input class='addCard__input' id="cardName" type="text" placeholder="add card name">
+            <input class='addCard__input' type="submit" value="Add Card">
```

Comment:
use a <button> here instead of input with type submit.

Reply:
use a <button> here instead of input with type submit.